content/applications/finance/accounting/bank/loans.rst

```diff
@@ -0,0 +1,72 @@
+:show-content:
+
+================
+Loans management
+================
+
+Odoo lets you manage your loans to have a comprehensive list of all loans undertaken by your company
+in order to maintain a holistic and forecasted view of upcoming due dates (e.g., cash forecast).
+Set up your amortization schedule—or import it—and let Odoo automatically handle monthly interest
+and principal adjustments such that your financial reports are always accurate with minimal effort.
+
+Create a new loan
+-----------------
+
+You can create a new loan by going to :menuselection:`Accounting --> Accounting --> Loans`. When
+creating a new loan, you can create your amortization schedule in three ways:
+
+- By importing it from a supported file;
+- By calculating it from a couple of values (e.g. the :guilabel:`Amount Borrowed`, the
+  :guilabel:`Duration`, etc.);
+- By manually filling in the lines of the schedule.
+
+In either case, 3 different fields are required for each line of the amortization schedule: the
+:guilabel:`Date`, the :guilabel:`Principal` and the :guilabel:`Interest`.
+
+The :guilabel:`Amount Borrowed`, :guilabel:`Interest` and :guilabel:`Duration` fields will be red
+if the sum of the lines does not match the totals amounts you have entered.
+
+Loan entries mechanism
+----------------------
+
+When the amount borrowed is credited to your bank account, it should be transferred to a long-term
+account (defined in the loan settings). Then, upon the validation of the loan, Odoo will create all
+the necessary journal entries such that you can always have a holistic and forecast view of upcoming
+due dates. The entire process is completely automated with a reclassification of long-term and
+short-term principal reclassification mechanism.
+
+For each line of the amortization schedule, Odoo creates the following entries:
+
+A payment entry on the same date that
+  - debits the principal amount from the long-term account;
+  - debit the interest amount from the expense account;
+  - credits the payment amount to the short-term account: this is the amount that will be
```

Comment:
```suggestion
  - debits the principal amount to the long-term account;
  - debits the interest amount to the expense account;
  - credits the payment amount to the short-term account: this is the amount that will be
```
I wasn't sure about if you debit *from* an account or *to* an account. "From" sounded weird to me, and the internet suggested that "to" is better here :)

Reply:
Are we sure about this one ? This sounds very weird to me. And I kinda "like" the fact that we `debit from`, then we `credit to` to better highlight the transfer.

Reply:
Alrighty, good points, requested changes have been pushed :)